kaspresso/src/main/kotlin/com/kaspersky/kaspresso/docloc/DocLocScreenshotCapturer.kt

```diff
@@ -1,6 +1,7 @@
 package com.kaspersky.kaspresso.docloc
 
 import com.kaspersky.kaspresso.device.screenshots.screenshotmaker.ScreenshotMaker
+import com.kaspersky.kaspresso.docloc.metadata.saver.MetadataSaver
```

Comment:
only import without usage?

Reply:
It is used in the consctructor. Before my changes MetadataSaver was in the same pachage as DocLocScreenshotCapturer. I made renamed it to the DefautMetadataSaver and extracted the interface MetadataSaver. Now it has to be imported